server/src/main/java/org/opensearch/action/StepListener.java

```diff
@@ -64,9 +65,10 @@
  *  }
  * }</pre>
  *
- * @opensearch.internal
+ * @opensearch.api
  */
 
+@PublicApi(since = "2.19.0")
```

Comment:
Just a thought here. Since `StepListener` has been used in Plugins before. Should we keep this is "1.0.0", basically since inception?

Reply:
Changed to 1.0.0